src/components/AgreementFilters/AgreementFilters.test.js

```diff
@@ -0,0 +1,104 @@
+import React from 'react';
+import '@folio/stripes-erm-components/test/jest/__mock__';
+import { renderWithIntl } from '@folio/stripes-erm-components/test/jest/helpers';
+import { MemoryRouter } from 'react-router-dom';
+// import { screen, waitForElementToBeRemoved } from '@testing-library/react';
+import { Accordion, Checkbox } from '@folio/stripes-testing';
+import translationsProperties from '../../../test/helpers';
+import { activeFilters, data } from './testResources';
+import AgreementFilters from './AgreementFilters';
+
+jest.mock('@folio/stripes-erm-components', () => ({
+   ...jest.requireActual('@folio/stripes-erm-components'),
+   OrganizationSelection: () => <div>OrganizationSelection</div>,
+}));
+
+jest.mock('@folio/stripes/smart-components');
```

Comment:
What does this mock do?

Reply:
It should be removed, it doesn't do anything